pkg/controllers/vdb/restart_reconciler.go

```diff
@@ -107,6 +107,15 @@ func (r *RestartReconciler) Reconcile(ctx context.Context, _ *ctrl.Request) (ctr
 		return ctrl.Result{}, err
 	}
 
+	// If create/revive db failed, we skip restarting the cluster for redoing create/revive db in vclusterops
+	if meta.UseVClusterOps(r.Vdb.Annotations) {
```

Comment:
Lets not make this specific to vcluster ops.

Reply:
As we discussed in the daily meeting, we will make this specific to vcluster ops and add a comment to explain.